lib/carconntwistprim.h

```diff
@@ -0,0 +1,134 @@
+// This file is part of libdsl, a library for heuristic graph search
+//
+// Copyright (C) 2004 Marin Kobilarov <marin@jhu.edu> and Subhransu Mishra <subhransu.kumar.mishra@gmail.com>
+//
+// This Source Code Form is subject to the terms of the Mozilla
+// Public License v. 2.0. If a copy of the MPL was not distributed
+// with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef DSL_CARCONNTWISTPRIM_H
+#define DSL_CARCONNTWISTPRIM_H
+
+#include "gridconnectivity.h"
+#include "cargrid.h"
+#include <vector>
+#include "carcost.h"
+#include "map.h"
+#include "carprimcfg.h"
+#include <Eigen/Dense>
+#include <Eigen/Geometry>
+
+namespace dsl {
+
+using SE2Prim = Eigen::Vector3d;
+using Vector1d = Eigen::Matrix<double, 1, 1>;
+
+/**
+ * Simple car connectivity using primitives. It enables the generation of arcs
+ * from a given SE(2) state. These are then used to connect two cells in SE(2)
+ * Each connection corresponds to an SE2Prim, i.e. a vector of SE(2) poses
+ *
+ * Author: Marin Kobilarov
+ */
+class CarConnTwistPrim: public GridConnectivity< Eigen::Vector3d, Eigen::Matrix3d, SE2Prim > {
```

Comment:
a better name, e.g. CarTwistConnectivity (best to avoid abbreviation if possible).